docs/concepts/whatisvpp/featurerich.rst

```diff
@@ -0,0 +1,70 @@
+.. _featurerich:
+
```

Comment:
This description of FD.io VPP features is a little-dated an un-organized.
I have sent you an over email a number of slides that gives much better positioning of FD.io